pom2.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
move to uat subdirectory?

Reply:
We need this in root folder to generate otf-uat jar from root directory as uat directory is no under src directory. We did to keep otf-uat logic separate from nucleus.

Reply:
I don't think that's right. You provide the path to this file in the codebuild definition, so you can just change that path there. If needed, you can of course copy the built artifacts somewhere else too, again using the codebuild definition